Exiled.CustomItems/Commands/Give.cs

```diff
@@ -96,7 +119,7 @@ public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out s
         /// </summary>
         private bool CheckEligible(Player player)
         {
-            return player.IsAlive && !player.IsScp && !player.IsCuffed && player.Inventory.items.Count < 8;
```

Comment:
This check should remain. You should never send an SCP player a fake syncvar making their client think they are human, as their client will expect to be able to do all human interactions that the server will ignore completely if the server thinks they are an SCP. Instead, you change the player's role to a human one, and then use player.ChangeApperance to make everyone elses client think they are still an SCP.

Reply:
Fixed on latest commit